_posts/phuong-my_do/2017-04-28-Business_analyst.md

```diff
@@ -0,0 +1,112 @@
+
+* **Review documentation** with the business stakeholders.
+
+* **Participate in solution design** (data modeling) with the software engineering team and organize walkthrough of the specifications with developers. 
+
+* For both activities described above, it is important to target the audience as Brian will not emphasize the same information. In one case, he wants to validate the impact to the users, in the other one he wants to break it down so the developers understand and can think of how this will impact their implementation.  
+
+* **Analyse and track issues**: the issue has to describe a user behaviour but explore the technical aspects of it. A good understanding of both functional and technical sides will help to better analyse the issues.
+
+* **UAT** (User Acceptance Testing):  Brian will define the user acceptance criteria of the application based on the requirements and business understanding and will perform quality assurance from a business perspective.
+
+* **Create user documentation and train users**.
+
+Other activities include: stakeholder analysis, vendor assessment (create a request for proposal), gap analysis, SWOT matrix (Strengths, Weaknesses, Opportunities, Threats), risk analysis, Ishikawa diagrams (or cause analysis), reverse engineering, focus group, MoSCoW (Must have, Should have, Could have, and Won't have but would like) and many other activities depending on the company needs.
+
+# Some comparison
+
+One path to become a business analyst is to start with IT studies. So between a developer and a BA, there’s only a small shift...
+
+| Category| Software Developer | Business Analyst  |
+| ----------- |-------------------| ----------------------|
+| Study     		| Software Engineering school | Software Engineering school another path could be business study with IT training down the road|
+| Ongoing learning | R&D on technologies, programming languages...      | Benchmark on company business and minimum understanding of software tools or best practice |
+| Challenges | Unclear and changing requirements and functional specifications coming from the business analysts |    Unclear and changing business requirements coming from the stakeholders |
+| Must have skills | Analytical, programing skills, problem-solving, willingness to research, teamwork... |    Analytical, documentation, listening, communication, negotiation, teamwork... |
+
+# Fun Facts
+* Gender equality: The business analysis is the IT job where gender equality is best represented (most of the time it’s half-half but I have worked in a BA department with 8 female BAs, 4 male BAs and a female manager). Note that this fact is purely based on experience with no large-scale survey behind that. (...but I think it’s true...)
```

Comment:
I don't think the parenthesis makes sense the first usage, but I wouldn't use them here at all.